lib/main.js

```diff
@@ -43,6 +43,11 @@ app.use('/micropub/:targetsite', micropub({
   tokenReference: function (req) {
     return (req.targetsite.token || []).concat(config.token);
   },
+  queryHandler: function (q, req) {
+    if (q === 'syndicate-to' && req.targetsite.syndicateTo) {
+      return { 'syndicate-to': req.targetsite.syndicateTo };
```

Comment:
If you want to try going the the full on deluxe version of things, then you could try to make an automated test for yourself in `test/` :)

I myself find that very handy when I don't have the code live anywhere or for any reason don't have a separate client that I can easily repeat the tests for. And the bonus is that I then know that I'll never break the code for that specific request in the future.

But I guess it can be a bit hard to figure out how to do that for this as this project hasn't really done any tests for `queryHandler`:s before, so if I were you I would skip it.